no-computer.md

```diff
@@ -11,41 +11,92 @@ _What need is met by your idea? Who are you helping? What is that person's
+would stir something up in me, it just clicked. Victor perfectly describes the
+tragedy of lost ideas, and I realized that my idea could easily become one of
+them, just because of lack of time. And worse -- how many people with similar
+ideas can't get their ideas out? How many stories will be lost? Anything that
+might prevent that from happening excites me.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Big stories emerging from small.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The syntax of the language consists of a set of short, unrelated stories, somewhere between a paragraph and one to two pages. All the stories are told from the first person. The narrators are not necessarily the same, but not necessarily different (this means that consistency is important; details from the different stories cannot contradict each other). There should be stories told about various times in the narrator's life, from childhood to adulthood. The stories should cover a range of emotions and character traits, and they should range from hilarious to harrowing.
+
+These stories can be printed on cards, from which the creator can choose a subset to put in any order to create the life story of the narrator, told in chunks. Even though the stories are not related, I believe that the language will have fluency as an emergent property; the more one reads of the assembled story, the more one will feel as if they know the narrator. The character will be developed as much through the contents of the stories as through the creator's judgment of what the narrator should choose to reveal about him or herself.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+A "program" running means someone reading the assembled work of the creator. As
+this happens, the reader will (hopefully) enjoy the individual stories. After
+having read enough of them, they will start to develop a picture of the
+narrator, hopefully learning something new about the character with each mini-
+story. Ideally, the stories would start to change in meaning depending on the
+context established by the particular set of stories chosen, and the order they
+are placed in. In this way, the reader may even see the same mini-stories in
+several different big stories, without realizing or caring about the
+repetition.
+
+The only errors I can think of are consistency errors, which are liable to
+happen with this kind of collage literature. These would have to be
+communicated through feedback to the creator and to the authors of the mini-
+stories which conflict. This is where the language needs an active community if
+it's to work. Given that, errors could be shared via this community.
```

Comment:
I to a certain degree think consistency errors are errors. We has humans have different opinions, and this applies to our opinions to individuals. One may have a completely different opinion, so having different views may not constitute a consistency error.